java/README.md

```diff
@@ -87,7 +87,7 @@ Other useful gradle developer commands:
 * `./gradlew :integTest:test` to run client examples
 * `./gradlew spotlessCheck` to check for codestyle issues
 * `./gradlew spotlessApply` to apply codestyle recommendations
-* `./gradlew :examples:run` to run client examples
+* `./gradlew :examples:run` to run client examples (make sure you have a running redis on port `6379`)
```

Comment:
@Yury-Fridlyand @acarbonetto When you're working on the readme file for GA - make sure to have a note before this line that guides what needed before running the example (is a standalone server is enough? what about the cluster example?)

Reply:
Got it!  I've added the action to our README.md update ticket.